src/backend/renderer/multigpu/mod.rs

```diff
@@ -1135,6 +1133,7 @@ where
                     frame
                         .clear([0.0, 0.0, 0.0, 0.0], &damage)
                         .map_err(Error::Target)?;
+                    frame.wait(&sync);
```

Comment:
We don't actually have to wait in this case. If this is a dmabuf imported by multiple device, sychronization is guaranteed to be handled internally.

Skipping this is important for the nvidia driver, which will wait correctly for the dmabuf, but can't import the native-fence.

Reply:
right, removed the call